### PR TITLE
Zihao/get homepage

### PR DESCRIPTION
**Pull request by ltroin**

### CODE REVIEW

File: src/screens/home/HomeScreen.js [hunk 1]
```diff
@@ -130,6 +161,21 @@ export const HomeScreen = ({ navigation }) => {
     justifyContent: "flex-start",
     marginLeft: modalMargin,
   };
+
+  const [chefs, setChefs] = useState([]);
```
Comment (anhanh11001) [inline src/screens/home/HomeScreen.js:165]: This should be removed. chefs list is already inside your state. You should use `chefs` from there

File: src/screens/home/HomeScreen.js [hunk 2]
```diff
@@ -130,6 +161,21 @@ export const HomeScreen = ({ navigation }) => {
...
+    dispatch(getChefs())
+      .unwrap()
+      .then((originalPromiseResult) => {
+        setChefs(originalPromiseResult);
+        // console.log(chefs);
+      })
+      .catch((rejectedValueOrSerializedError) => {
+        console.log(rejectedValueOrSerializedError);
+      });
+  }, []);
```
Comment (anhanh11001) [inline src/screens/home/HomeScreen.js:177]: No need to do unwrap here, just use chefs from the state

File: src/redux/reducers/chefReducer.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import { getChef } from "../../api/auth";
+
+export class ChefState {
+  constructor(state) {
+    this.state = state;
+  }
+  static ChefWaiting = new ChefState("waiting Restaurants...");
```
Comment (anhanh11001) [inline src/redux/reducers/chefReducer.js:8]: Remove this ChefWaiting. It 's the same as RequestingChefs

**Comment by ltroin** [comment]
changes done

**Review by anhanh11001 — APPROVED** [review]

**Pull request merged**